kafka/src/main/java/io/micronaut/configuration/kafka/bind/batch/BatchConsumerRecordsBinderRegistry.java

```diff
@@ -65,8 +65,7 @@ public BatchConsumerRecordsBinderRegistry(ConsumerRecordBinderRegistry consumerR
 
             return Optional.of((context, consumerRecords) -> {
                 for (ConsumerRecord<?, ?> consumerRecord : consumerRecords) {
-                    Optional<ArgumentBinder<?, ConsumerRecord<?, ?>>> binder = consumerRecordBinderRegistry.findArgumentBinder((Argument) argument, consumerRecord);
-                    binder.ifPresent(b -> {
+                    consumerRecordBinderRegistry.findArgumentBinder(argument).ifPresent(b -> {
```

Comment:
does not have to look for a consumer record anymore?

Reply:
It seems that the second parameter has become redundant and `findArgumentBinder(Argument<T>, S)` [has been replaced](https://github.com/micronaut-projects/micronaut-core/blob/4.0.x/core/src/main/java/io/micronaut/core/bind/ArgumentBinderRegistry.java#L63) with just `findArgumentBinder(Argument<T>)`.